README.md

```diff
@@ -69,6 +69,20 @@ Run the following commands:
 1. `./gradlew createDB -PdbName=fineract_default`
 1. `./gradlew devRun`
 
+Fineract is now running, and will be listening for API requests on port 8443 by default.
+
+Confirm Fineract is ready to receive requests with the endpoint:
```

Comment:
"Confirm Fineract is ready with, for example:" makes more sense here. An example API request using curl is what follows, not an endpoint.